jenkins/Jenkinsfile

```
@@ -23,6 +30,19 @@ pipeline {
   }
   
   stages {
+    stage('Prepare'){
+      steps {
+        container('node'){
+          dir("${stage_dir}/conf") {
+            withVault([configuration: configuration, vaultSecrets: secrets]) {
+              sh """
+                jq '.maps.accessToken = "${env.WIKIMEDIA_ACCESS_TOKEN}"' app.json > app.json.tmp && cp app.json.tmp app.json
```

1. We need to adapt to the previous change (whatever you choose as a name of env var)
2. I think it would be better to use `mv` instead of `cp` not to keep the `app.json.tmp` file in the RPM package (and then after installation on the Cloudify Manager VM)

```suggestion
                jq '.maps.accessToken = "${env.MAPS_ACCESS_TOKEN}"' app.json > app.json.tmp && mv app.json.tmp app.json
```

---

sounds good